content/community/contents.lr

```diff
@@ -12,6 +12,10 @@ CC has an active Slack community open to the public. [Sign up here](http://slack
 
 If you're looking to talk to someone on CC's tech team, we are all in the Americas and someone is usually around on Slack between 11:00 and 23:00 UTC.
 
+For information regarding joining a channel on Slack, [look here](https://get.slack.help/hc/en-us/articles/205239967-Join-a-channel).
+
+Note that channels like `#general` on Slack consist of a fair amount of users. While it is a channel for general-purpose discussions, please take into consideration that a post on such a channel might notify thousands of users. 😄
```

Comment:
could you change "might" to "will"? otherwise, this looks great.

Reply:
Done!